### ISSUE TITLE
References (in subdirectory) not found for HTML book

### ISSUE THREAD

**Issue by nclJoshCowley** [quarto-dev/quarto-cli#3949]
### Bug description

Putting `references.bib` (or a csl file, as I was attempting) in a subdirectory Quarto does not find it for HTML only.

For a reprex, create a Quarto book from template (say, `quarto create-project mybook --type book`)
Then place the .bib fiile in a new directory called `subdir` then modify `_quarto.yml`, line 15.

That should create the following key files:

#### index.qmd

```
# Preface {.unnumbered}

See @knuth84 for additional discussion of literate programming.
```

#### references.qmd

```
# References {.unnumbered}

::: {#refs}
:::
```

#### _quarto.yml

```
project:
  type: book

book:
  title: "issue3949"
  author: "Jane Doe"
  date: "12/01/2023"
  chapters:
    - index.qmd
    - references.qmd

bibliography: subdir/references.bib

format:
  html:
    theme: cosmo
  pdf:
    documentclass: scrreprt
````

#### subdir/references.bib

```
@article{knuth84,
  author = {Knuth, Donald E.},
  title = {Literate Programming},
  year = {1984},
  issue_date = {May 1984},
  publisher = {Oxford University Press, Inc.},
  address = {USA},
  volume = {27},
  number = {2},
  issn = {0010-4620},
  url = {https://doi.org/10.1093/comjnl/27.2.97},
  doi = {10.1093/comjnl/27.2.97},
  journal = {Comput. J.},
  month = may,
  pages = {97–111},
  numpages = {15}
}
```

If I render to PDF this works fine, Rendering to HTML leads to 

> File subdir.bib not found in resource path
> ERROR: File subdir.bib not found in resource path

Dashes seem to preserve some of the name as `bibliography: subdir/references-2.bib` leads to 

> File subdir-2.bib not found in resource path

This is on Quarto version 1.2.313,

<details><summary>RStudio Version</summary>
RStudio 2022.12.0+353 "Elsbeth Geranium" Release (7d165dcfc1b6d300eb247738db2c7076234f6ef0, 2022-12-03) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2022.12.0+353 Chrome/102.0.5005.167 Electron/19.1.3 Safari/537.36
</details>

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by cscheid** [comment]
Can you please actually share a complete, fully reproducible example? That's a .qmd and all needed associated files. Thank you.

**Comment by nclJoshCowley** [comment]
Updated the main post with a fully reproducible example.

**Comment by cderv** [comment]
Yes this is a known issue - duplicates of #3944 I believe. #3952 will fix it.

**Comment by cderv** [comment]
This should now be fixed in v1.3.121

**Comment by nclJoshCowley** [comment]
Aploglies for the duplicate issue.

Just tested with latest pre-release version (1.3.124) and works for above example and my use-case, thanks.
